src/components/CreateCampaignFromTemplateDialog.tsx

```diff
@@ -18,6 +18,7 @@ export interface CreateCampaignFromTemplateDialogProps {
   organizationId: string;
   open: boolean;
   onClose?: () => Promise<void> | void;
+  preselectedTemplate?: TemplateCampaignFragment;
```

Comment:
Nitpick: possibly rename this to `template` to stay consistent with `CreateCampaign`